spec/system/work_wizard_spec.rb

```diff
@@ -96,7 +98,6 @@
     click_on "Grant License and Complete"
     page.driver.browser.switch_to.alert.accept
     expect(page).to have_content("5-10 business days")
-    expect(page).to have_content(work.title)
```

Comment:
We don't display the title in https://github.com/pulibrary/pdc_describe/blob/main/app/views/works_wizard_submission_complete/show.html.erb (but how come this passed sometimes?)